api/migrations/Version20230420133134.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230420133134 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
Name your migration, something like 'Initialize the database'. In this project you won't have much migrations but in a big project you could have more than 100 and it's easier to read the description instead of the up and down methods body.